### ISSUE TITLE
systemd unit file issues

### ISSUE THREAD

**Issue by mattaezell** [xcat2/goconserver#24]
I see the following messages in the system journal:
```
Feb 08 10:09:58 mgmt1.peak.olcf.ornl.gov systemd[1]: [/usr/lib/systemd/system/goconserver.service:8] Unknown lvalue 'killMode' in section 'Service'
Feb 08 10:09:58 mgmt1.peak.olcf.ornl.gov systemd[1]: [/usr/lib/systemd/system/goconserver.service:9] Unknown lvalue 'After' in section 'Service'
```
The `After` parameter should go in the `[Unit]` section, not `[Service]`
`killMode` needs to be `KillMode`

We are also seeing shutdown hangs that appear to be because goconserver is writing to a NFS share.  Still looking into the cause...

**Comment by robin2008** [comment]
To fix the `After` issue in #46

**Comment by robin2008** [comment]
Need to update xcat-dep accordingly in 2.14.3
